api/docs/schemas/groups/events/groupeLeaveEvent.ts

```diff
@@ -0,0 +1,49 @@
+import { HandlerDefinition } from '../../../../types/typing';
+
+const schema: HandlerDefinition = {
+  path: '/groups/event/leave',
+  post: {
+    operationId: 'groupLeaveEvent',
+    description: 'A group leaves an event',
+    security: [{ BearerAuth: [] }],
+    tags: ['groups'],
+    requestBody: {
+      required: true,
+      content: {
+        'application/json': {
+          schema: {
+            type: 'object',
+            properties: {
+              groupId: { type: 'string' },
+              eventId: { type: 'string' },
+            },
```

Comment:
We talked about it few days ago but now i do think it's better to do a path like this

`/groups/{groupId}/events/{eventId}/leave`, it's way more clear when we read the path of the route to undestand what it does !

Reply:
I think it's better because the path of a route need to defined an entity, for example : 

- /musicians defines all the musicians
- /musicians/{musicianId} defines one musician

so it is clear than `/groups/{groupId}/events/{eventId}/leave` defines the fact that the group `groupId` want to leave the event  `eventId`